app/controllers/api/autocomplete_controller.rb

```diff
@@ -0,0 +1,11 @@
+module Api
+  class AutocompleteController < ApplicationController
+    before_action :authenticate_user!
```

Comment:
Totally. For now, it's good enough like this, but we will probably have to change it later on. I don't know much details about the web API that's needed in the project (public-facing, only for web app or available for other private clients, etc). We will need to talk about this.